documentation/src/main/asciidoc/topics/ref_ports_protocols.adoc

```diff
@@ -0,0 +1,52 @@
+[id='ports_protocols-{context}']
+= Ports and Protocols
+{brandname} uses the following ports and protocols to allow network traffic
+between remote endpoints and clustered nodes:
```

Comment:
IMO we should separate the ports used to connect clients to remote endpoints (where we only need the port to be open on the server) and the ports used internally (where we need every node to connect to every other node).

I'd also change the text to say "TCP and UDP ports" instead of "ports and protocols", because we can't mix-and-match, and add "default" somewhere, because all of them can be changed by the user.